flopy/mf6/utils/generate_classes.py

```diff
@@ -194,8 +197,15 @@ def generate_classes(
     print("  Deleting existing mf6 classes")
     delete_mf6_classes()
 
-    print("  Creating mf6 classes using the definition files")
-    create_packages()
+    # convert dfns to toml.. when we
+    # do this upstream, remove this.
+    _TOML_PATH.mkdir(exist_ok=True)
+    from modflow_devtools.dfn2toml import convert as dfn2toml
+    dfn2toml(_DFN_PATH, _TOML_PATH)
```

Comment:
this is where the toml conversion happens, and below we load the definitions in the new toml format.